README.md

```diff
@@ -7,6 +7,8 @@ mode, if you `require` an Elm module from a Webpack entry point, not only will
 that `.elm` file be watched for changes, but any other Elm modules it imports will
 be watched for changes as well.
 
+Checkout [Jetpack] as an alternative to Webpack!
+
```

Comment:
I think it's a little wierd to reference alternative build systems in a plugin for a particular build system. :/

Reply:
I thought it was interesting as an elm-focused build system, and I've seen other projects do similar project shoutouts in their readme's. I can drop this if you think it's inappropriate. Otherwise, I can actually add the link to https://github.com/NoRedInk/jetpack

Reply:
I don't have strong feelings either way. Just found it a bit strange, is all. Ok then, let's merge :)

Reply:
Done! Do you want to take care of the release for npm?

Reply:
Done